### ISSUE TITLE
Rakudo didn't install properly

### ISSUE THREAD

**Issue by ghost** [its-pointless/gcc_termux#129]
![2021-09-03-100427_1600x900_scrot_crop](https://user-images.githubusercontent.com/26301853/132010174-6a4ca5fe-0638-4ea5-90db-7de64ec46989.png)

u0_a74@localhost ~/rakudo [100]> pkg show rakudo -a
Package: rakudo
Version: 2020.05
Maintainer: Termux members @termux
Installed-Size: 37.7 MB
Depends: moarvm
Homepage: https://rakudo.org
Download-Size: 5062 kB
APT-Manual-Installed: yes
APT-Sources: https://its-pointless.github.io/files/24 termux/extras arm Packages                                                           
Description: Perl 6 implementation on top of Moar virtual machine

Package: rakudo
Version: 2020.01-1
Maintainer: Fredrik Fornwall @fornwall
Installed-Size: 93.1 MB
Depends: moarvm
Homepage: https://rakudo.org
Download-Size: 10.9 MB
APT-Sources: https://its-pointless.github.io/files/24 termux/extras arm Packages                                                           
Description: Perl 6 implementation on top of Moar virtual machine

u0_a74@localhost ~/rakudo> raku
CANNOT LINK EXECUTABLE "raku": cannot locate symbol "ffi_type_double" referenced by "/data/data/com.termux/files/usr/lib/libmoar.so"...    
u0_a74@localhost ~/rakudo> raku --version
CANNOT LINK EXECUTABLE "raku": cannot locate symbol "ffi_type_double" referenced by "/data/data/com.termux/files/usr/lib/libmoar.so"...    
u0_a74@localhost ~/rakudo> raku --help
CANNOT LINK EXECUTABLE "raku": cannot locate symbol "ffi_type_double" referenced by "/data/data/com.termux/files/usr/lib/libmoar.so"...    
u0_a74@localhost ~/rakudo> uname -a
Linux localhost 3.4.42-g3d041de #1 SMP PREEMPT Sat Dec 24 19:56:29 PST 2016 armv7l Android                                                 
u0_a74@localhost ~/rakudo>

i just run the script https://github.com/its-pointless/gcc_termux/blob/master/setup-pointless-repo.sh
and then `pkg install rakudo`

**Comment by its-pointless** [comment]
yeah probably best to compile it on device for now. I will take a look at some point next few weeks.

**Comment by rrodrigueznt** [comment]
> yeah probably best to compile it on device for now. I will take a look at some point next few weeks. 

Please, could you point us in the right direction to get it compiled on device? Rakudo is notin the termux-packages folder. Please, could you give us a link to the best documentation you are aware off to get this done? Thanks!
